content/quickstart.md

```diff
@@ -7,29 +8,72 @@ tocopen: true
 
 # Prerequisites
 
-Under construction.
+1- OpenSeeFace tracking: webcam

```

Comment:
In markdown, you can get an ordered/unordered list by doing either

"1."

or

"-"

You should avoid brute forcing lists like this. Note that sentences in lists should still start with a capital letter but periods can be optional as long as you consistently use/omit them